docs/engines/01_available_engines.rst

```diff
@@ -0,0 +1,124 @@
+
+Available Engines
+=================
+
+
+
+The tables below give a high-level overview of the planning engines integrated with the UP.
+
+Action-Based Planning
+^^^^^^^^^^^^^^^^^^^^^
+
+.. list-table:: 
+
+  * - Engine
+    - Operation modes
+    - Classical
+    - Numeric
+    - Temporal
+    - Metrics
+  * - `Fast-Downward`_
+    - OneShot, Anytime
+    - 🗸
```

Comment:
If I build the docs locally, this symbol is not rendered well. Am I missing any requirements?

Reply:
Strange, it is just a UTF-8 character. Probably, a font problem. I'll change it to `Y` for better compatibility (but it looks less cool)